inc/main.php

```diff
@@ -69,6 +69,9 @@ public function __construct() {
 	public static function instance() {
 		if ( null === self::$_instance ) {
 			add_filter( 'themeisle_sdk_products', array( __CLASS__, 'register_sdk' ) );
+			add_filter( 'ti_custom_deactivate_cleanup', function ($key) {
```

Comment:
Can we have separate functions just to keep everything the looking the same instead of anonimus functions?